qgreenland/config/datasets/nunagis_protected_areas.py

```diff
@@ -1,13 +1,18 @@
-from qgreenland.models.config.asset import OgrRemoteAsset
+from qgreenland.models.config.asset import CommandAsset
 from qgreenland.models.config.dataset import Dataset
 
 
 nunagis_protected_areas = Dataset(
     id='nunagis_protected_areas',
     assets=[
-        OgrRemoteAsset(
+        CommandAsset(
             id='only',
-            query_url='https://kort.nunagis.gl/server/rest/services/Hosted/Bird_important_areas/FeatureServer/0/query?f=json&where=true&outFields=*&orderByFields=fid0+ASC',
+            args=[
```

Comment:
We should probably create a helper for this since the same args (aside from the URL) is used for the `nunagis_pop2019_municipalities` asset.

Reply:
We don't have any other dataset helpers yet. I don't think there's any reason not to, though!